templates/bundles/PrestaShopBundle/Admin/Configure/ShopParameters/preferences.html.twig

```diff
@@ -0,0 +1,77 @@
+{#**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ *#}
+{% extends '@PrestaShop/Admin/layout.html.twig' %}
+{% trans_default_domain "Admin.Shopparameters.Feature" %}
+{% form_theme generalForm '@PrestaShop/Admin/TwigTemplateForm/prestashop_ui_kit.html.twig' %}
+
+{% block content %}
```

Comment:
Are you sure this file is at the right place? I believe putting templates in the `templates/bundles` folder was actually not necessary or even a bad idea but I don't remember the details exactly
@atomiix does it ring a bell?

Reply:
Besides this file seems like a copy of `src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/preferences.html.twig` Maybe from an previous implementation since this PR is quite old So maybe it should simply be removed?

Reply:
You're right @jolelievre, we should not use the `./templates` folder for PrestaShopBundle templates. It should go to `src/PrestaShopBundle/Resources/views/`